libcontainer/cgroups/systemd/v1.go

```diff
@@ -17,10 +17,17 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
-type LegacyManager struct {
+type legacyManager struct {
 	mu      sync.Mutex
-	Cgroups *configs.Cgroup
-	Paths   map[string]string
+	cgroups *configs.Cgroup
+	paths   map[string]string
+}
+
+func NewLegacyManager(cg *configs.Cgroup, paths map[string]string) *legacyManager {
```

Comment:
ditto: should return a public interface

Reply:
Yeah, I have noticed the discrepancy. Apparently it's OK to return a private struct as long as it conforms to the interface. Anyway, I'll unify to return cgroups.Manager everywhere

Reply:
fixed

Reply:
`NewLegacyManager` doesn't seem fixed

Reply:
fixed